src/components/ChatInfo/TextBackground.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from "react";
+
+interface IProps {
+  width: number;
+  height: number;
+  deg?: number;
+  left: number;
+  bottom?: number;
+  right?: number;
+  top?: number;
+  zIndex?: number;
```

Comment:
I think only optional props should be zIndex, if deg, bottom, right and top are optional we need default value